pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -1760,12 +1759,7 @@ var _ reconciler.ConfigStore = (*testConfigStore)(nil)
 
 func TestMetricsReporter(t *testing.T) {
 	pa := kpa(testNamespace, testRevision)
-	wantTags := map[string]string{
```

Comment:
Tests in this file needs to be changed to use `metricstest.AssertMetric` as well.

Reply:
Done.